app/src/main/java/org/fossasia/openevent/events/ErrorDialogEvent.java

```diff
@@ -0,0 +1,59 @@
+package org.fossasia.openevent.events;
+
+import android.util.Log;
+
+import org.fossasia.openevent.OpenEventApp;
+import org.fossasia.openevent.api.APIClient;
+
+import retrofit.RetrofitError;
+
+/**
+ * Created by shivenmian on 28/12/15.
+ */
+public class ErrorDialogEvent {
+
+    public String ErrorType;
+    public String ErrorDesc;
+    public static final String TYPE = "RetrofitError Type";
+    public static final String ERROR_CODE = "Error Code";
+
+    public ErrorDialogEvent(String s1, String s2){
+        this.ErrorType = s1;
+        this.ErrorDesc = s2;
+    }
+
+    public void ErrorHandlerFunction(RetrofitError cause, ErrorDialogEvent event){
+
+        if (cause.getKind().equals(RetrofitError.Kind.CONVERSION)) {
```

Comment:
And I get 

![image](https://cloud.githubusercontent.com/assets/547235/12030961/5a377a04-ae41-11e5-890a-91777ea7e371.png) 
when I plug in http://www.mocky.io/v2/5682128c120000993893a2c1

I think it works, do you have more counterexamples?